Week1/solution4.py

```diff
@@ -0,0 +1,18 @@
+'''
+Write a python script to enter any number, if it is integer number, then check its armstrong or not.
+Print appropriate message if it is not palindrome.
+'''
+n=float(input("Enter a number:"))
+if int(n)==n:

```

Comment:
explain this statement

Reply:
variable n is used to take both float or int value from user. The value them passes through if condition to check whether variable n is int or not. If int(n) = n is true, the value is integer, else the value is float.